app/routes/docs/newproducers.mdx

```diff
@@ -0,0 +1,97 @@
+---
+handle:
+  breadcrumb: New Producers
+---
+
+import {
+  ProcessList,
+  ProcessListItem,
+  ProcessListHeading,
+  SummaryBox,
+} from '@trussworks/react-uswds'
+import { GitHubLabel } from '~/components/GitHubLabel'
+
+# Setting up New GCN Notice Producers
+
+The following steps guide new instrument, mission, or observatory producers
+into setting up new notices streams that are distributed to the user community
+via [Kafka](faq#what-is-kafka). This process requires interactions
+with the [GCN Team](https://heasarc.gsfc.nasa.gov/cgi-bin/Feedback?selected=kafkagcn)
+to enable these accounts and topics are setup on the GCN Kafka broker,
+and we are happy to work together to help you define schema utilizing our
+core JSON schema.
+
+## Steps to Start Publishing Alerts
+
+<ProcessList>
+   <ProcessListItem>
+     <ProcessListHeading type="h3">
+       Team Member Accounts
+     </ProcessListHeading>
+   - Make sure that your team member(s) who will implement GCN notices in your pipeline
+   has signed in (at least once) to https://test.gcn.nasa.gov and https://gcn.nasa.gov.
+   - Send the [GCN Team](https://heasarc.gsfc.nasa.gov/cgi-bin/Feedback?selected=kafkagcn)
+   a list of email address(es) of all the team members who should have the
+   producer access to the <code><i>mission_name</i>.\*</code> topics.
+   </ProcessListItem>
+   <ProcessListItem>
+      <ProcessListHeading type="h3">
+         Defining Kafka Topics and JSON Schema
+      </ProcessListHeading>
+      The preferred format for formatting of the data sent via GCN Kafka is JSON.
+      We have designed a set of [core schema](https://github.com/nasa-gcn/gcn-schema/tree/master/core-schema)
```

Comment:
I don't think that the unified schema is ready for prime time yet. We haven't even checked that they are valid JSON Schema.

Reply:
validation is mostly addressed.  I removed this, but we do need to get the schema ready to go.